benchmark/fs/bench-readSync.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const common = require('../common');
+const fs = require('fs');
+const tmpdir = require('../../test/common/tmpdir');
+tmpdir.refresh();
+
+const bufferSize = 1024;
+const sectorSize = 512;
+const tmpfile = { name: tmpdir.resolve(`.existing-file-${process.pid}`),
+                  len: bufferSize * 1e4 };
```

Comment:
The n should not be hard-coded here, can you move this into the benchmark main and calculate len as `bufferSize * n`?

Reply:
Thanks, I moved it.